integration-test/src/test/java/org/sagebionetworks/IT500SynapseJavaClient.java

```diff
@@ -1099,4 +919,39 @@ public void testGetEntityChildrenProjects() throws SynapseException{
 	public void testLookupEntity() throws SynapseException {
 		assertEquals(dataset.getId(), synapseOne.lookupChild(project.getId(), dataset.getName()));
 	}
+	
+	
+	/*
+	 * Test deprecated services
+	 * These test can be removed when the services are removed
+	 */
+	
+	@Test
```

Comment:
Note that you could use the annotation instead in this case.
Like:
@Test (expected= SynapseDeprecatedServiceException.class)